protocol/messenger.go

```diff
@@ -1748,40 +1752,41 @@ func (m *Messenger) Init() error {
 
 	// Get chat IDs and public keys from the existing chats.
 	// TODO: Get only active chats by the query.
+	logger.Info("imotta: m.persistence.Chats")
 	chats, err := m.persistence.Chats()
 	if err != nil {
 		return err
 	}
+
+	logger.Info("imotta: begin chat loop", zap.Any("numberOfChats", len(chats)))
+	communityInfo := make(map[string]*communities.Community)
```

Comment:
This is the line that matters, the map is extracted out of the loop.

**All log calls will be removed before merge**, but I wanted to keep them until mobile QA approve the mobile build. In case something unexpected is reported, QAs will be able to share more rich logs.